python_modules/libraries/dagster-dg/dagster_dg/component.py

```diff
@@ -104,8 +105,17 @@ def _load_entry_point_components(
     if dg_context.has_cache:
         cache_key = dg_context.get_cache_key("plugin_registry_data")
         raw_registry_data = dg_context.cache.get(cache_key)
+        if raw_registry_data is not None:
+            # If we encounter a deserialization error (which can happen due to version skew),
+            # just treat the cache as invalid and fetch the data again.
+            try:
+                return _parse_raw_registry_data(raw_registry_data)
+            except DeserializationError:
+                raw_registry_data = None  # invalid
```

Comment:
should we log something here, even if at a more verbose level, for debugging purposes?

Reply:
I have a PR upstack that does this